ideascube/serveradmin/management/commands/csv2pkg.py

```diff
@@ -13,8 +16,29 @@
 Optional columns are "lang", "preview", "kind", "tags".
 Attributes "path" and "preview" (if provided) must be path
 (relative to csv directory) to existing files.
+
+csv2pkg also generates a yaml to add to the catalog.
+Some attributes are automically calculated from the generated package.
+Other attributes may be provided as command arguments.
 """
 
+def generate_catalog_yaml_metadata(path, _type, package_id, name,
+                                   description, language):
```

Comment:
The function doesn't just create a dictionary.
It also make some operation on the file to get its sha and  size.

And I like to separate code in different function when the code is about different stuff.
The problem here come from the function definition who take to much arguments and so seems to just put all the given arguments in a dict. (And seems to be a code duplication with the caller code that extract information from the options dict)

I will rewrite the function to take only the `path` and the `options` arguments.